FHEM/14_SD_UT.pm

```diff
@@ -69,13 +66,13 @@ use warnings;
 
 sub SD_UT_Initialize($) {
 	my ($hash) = @_;
-	$hash->{Match}		= "^[P|u](29|30|79|81|83)#.*";
+	$hash->{Match}		= "^[P|u](29|30|81|83)#.*";
```

Comment:
Pack die "oder" Protokolle besser mal in eine non capturing group:

`"^[P|u]?:(29|30|81|83)#.*`